package.json

```diff
@@ -20,7 +21,10 @@
     "@angular/platform-browser": "^6.1.0",
     "@angular/platform-browser-dynamic": "^6.1.0",
     "@angular/router": "^6.1.0",
+    "angular-highcharts": "^6.2.6",
+    "angular2-highcharts": "^0.5.5",
```

Comment:
Why do we need both highcharts libraries?

Reply:
done